packages/block-editor/src/components/use-display-block-controls/index.native.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { hasBlockSupport } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { useBlockEditContext } from '../block-edit/context';
+import { store as blockEditorStore } from '../../store';
+
+export default function useDisplayBlockControls() {
+	const { isSelected, clientId, name } = useBlockEditContext();
+	return useSelect(
+		( select ) => {
+			const { getBlockName, getBlockRootClientId } = select(
+				blockEditorStore
+			);
+
+			const parentId = getBlockRootClientId( clientId );
+			const parentBlockName = getBlockName( parentId );
+
+			const displayControls = hasBlockSupport(
+				parentBlockName,
+				'__experimentalExposeControlsToChildren',
+				true
+			);
```

Comment:
@guarani, thank you for this! You're right I jumped to the wrong assumption about the purpose of this block support 🤦‍♀️ but agree that its existence gives hope that we can introduce other experimental supports for customizing inner blocks. 

I've updated this PR to use `__experimentalDisplayChildBlockControls` instead of `experimentalExposeControlsToChildren`, which also works for our end goal in my testing.  

It'd be great to hear your thoughts!

Reply:
Introducing a new block supports sounds like a good idea here 👍 
I'm marking this thread resolved.